XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -2077,15 +2077,30 @@ public enum CustomComboPreset
     [CustomComboInfo("Kasha Combo Option", "Start the Kasha combo chain with Shifu instead of Hakaze.", SAM.JobID)]
     SamuraiKashaOption = 3417,
 
-    [IconsCombo([SAM.Mangetsu, UTL.ArrowLeft, SAM.Fuga])]
+    [IconsCombo([SAM.Mangetsu, SAM.Oka, UTL.ArrowLeft, SAM.Fuga, SAM.Fuko])]
     [SectionCombo("Area of Effect")]
-    [CustomComboInfo("Mangetsu Combo", "Replace Mangetsu with its combo chain.", SAM.JobID)]
-    SamuraiMangetsuCombo = 3404,
+    [CustomComboInfo("Oka/Mangetsu Combo", "Replace Oka and Mangetsu with their combo chain.", SAM.JobID)]
+    SamuraiAoECombo = 3404,
 
     [IconsCombo([SAM.Oka, UTL.ArrowLeft, SAM.Fuga])]
     [SectionCombo("Area of Effect")]
-    [CustomComboInfo("Oka Combo", "Replace Oka with its combo chain.", SAM.JobID)]
-    SamuraiOkaCombo = 3405,
+    [AccessibilityCustomCombo]
+    [CustomComboInfo("Auto-AoE", "Replace Oka and Mengutsu with whichever one you don't have the Sen for yet.  If you have both or neither Sen, neither one will be replaced.", SAM.JobID)]
+    SamuraiAutoAoEFeature = 3423,
+
+    [IconsCombo([SAM.Oka, UTL.ArrowLeft, SAM.Fuga])]
+    [SectionCombo("Area of Effect")]
+    [AccessibilityCustomCombo]
+    [ParentCombo(SamuraiAutoAoEFeature)]
+    [CustomComboInfo("Auto-AoE Buff Upkeep", "Also replace Oka and Mangutsu with each other in order to upkeep the buffs they apply, provided you either have neither of the two Sen, or already have both.  When you have neither or both of the Sen, both Oka and Mangetsu will be replaced with whichever one grants the buff that currently has the shortest duration on you (or is currently absent).  If both buffs are absent, neither ability will be replaced", SAM.JobID)]
```

Comment:
It's Mangetsu too*

Reply:
Geez, I just yolo'd the spelling on this one >.>